include/zephyr/usb/usbd.h

```diff
@@ -141,6 +141,18 @@ struct usbd_ch9_data {
 	uint8_t alternate[USBD_NUMOF_INTERFACES_MAX];
 };
 
+/**
+ * @brief USB device speed
+ */
+enum usbd_speed {
+	/** Device supports or is connected to a full speed bus */
```

Comment:
no low speed? (guess I'm missing some context here)

Reply:
I don't think there is any hardware in Zephyr that even supports Low-Speed as device, so it is not really possible to test if someone did implement Low-Speed at the stack side (the same can be said about the SS enum value here though).

Reply:
ok fair enough, thanks for clarifying